### ISSUE TITLE
Is this a work in progress?

### ISSUE THREAD

**Issue by jinny1208** [yangdongchao/SoundStream#1]
Hello! Do you have any plans of uploading the code for SoundStream and Encodec in the near future? Thanks in advance!

**Comment by amitaie** [comment]
Hey, I would also like to know, thank you!

**Comment by lizeyu519** [comment]
@yangdongchao can you share the encodec training code if possible?

**Comment by yangdongchao** [comment]
> Hello! Do you have any plans of uploading the code for SoundStream and Encodec in the near future? Thanks in advance!

Yes, we plan to provide the overall code and technical report in the next month.
